main.tf

```diff
@@ -9,11 +9,11 @@ resource "aws_security_group" "default" {
 resource "aws_security_group_rule" "egress" {
   count             = module.this.enabled ? 1 : 0
   type              = "egress"
-  description       = "Allow all egress traffic"
-  from_port         = 0
-  to_port           = 0
-  protocol          = "-1"
-  cidr_blocks       = ["0.0.0.0/0"]
+  description       = "Allow outbound traffic from CIDR blocks"
+  from_port         = var.egress_port
+  to_port           = var.egress_port
```

Comment:
please make these separate variables

Reply:
new variables `egress_source_port` & `egress_dest_port` are added.
kindly check, thanks @milldr

Reply:
@milldr

Reply:
from_port and to_port specify the range of ports allowed, not the destination and source ports.

Reply:
Hi @SebastianMCarreira , can you help to elaborate more?
The terraform resource _["aws_security_group_rule"](https://registry.terraform.io/providers/hashicorp/aws/3.42.0/docs/resources/security_group_rule)_ has inputs _from_port_  and  _to_port_ , which behave like range of ports afaik. 

from this PR we propose to adjust the docdb egress port range to be dynamic, so the docdb can handle other than `from_port=0`,`to_port=0`, and `protocol="-1"` (too wide),

For example, a user can do this.
```
  from_port      = var.egress_source_port -> can be filled with 0
  to_port        = var.egress_dest_port --> can be filled with 65535
```

Are these new variable names needed to be changed?
`egress_source_port` & `egress_dest_port`

Thank you for your concern

Reply:
@milldr 
PR updated. Modified new variables naming. Kindly review, thanks.

_make init_ ✔️ 
_make readme_ ✔️

Reply:
@haidargit looks like there were changes after `make readme` - ([failed test](https://github.com/cloudposse/terraform-aws-documentdb-cluster/actions/runs/5557065012/jobs/10227284934?pr=54))

please run one more time

Reply:
Thanks @milldr 

`make init` & `make readme` have been re-run
PR updated, kindly check again